{{ cookiecutter.package_name|slugify }}/.github/workflows/test.yml

```diff
@@ -9,48 +9,46 @@ on:
 
 jobs:
 
-  test:
-
+  build:
     runs-on: ubuntu-latest
-
-    strategy:
-
-      matrix:
-        python-version: ["{{ cookiecutter.python_version }}"]
```

Comment:
Although I'm all for using Docker, we do have to be aware that this PR removes the ability to test in a matrix, which is a bit of a downgrade for Python packages (for Python applications that shouldn't matter too much).